cpp/example_code/dynamodb/create_table.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+This file is licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License. A copy of
+the License is located at
+
+http://aws.amazon.com/apache2.0/
+
+This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+#include <aws/core/Aws.h>
+#include <aws/core/utils/Outcome.h> 
+#include <aws/dynamodb/DynamoDBClient.h>
+#include <aws/dynamodb/model/AttributeDefinition.h>
+#include <aws/dynamodb/model/CreateTableRequest.h>
+#include <aws/dynamodb/model/KeySchemaElement.h>
+#include <aws/dynamodb/model/ProvisionedThroughput.h>
+#include <aws/dynamodb/model/ScalarAttributeType.h>
+#include <iostream>
+
+
+/**
+* Deletes a cloud watch alarm, based on command line input
+*/
+int main(int argc, char** argv)
+{
+	const Aws::String USAGE = "\n" \
+		"Usage:\n"
+		"    create_table <table> <optional:region>\n\n"
+		"Where:\n"
+		"    table - the table to create\n"
+		"    region- optional region\n\n"
+		"Example:\n"
+		"    create_table HelloTable us-west-2\n";
+
+	if (argc < 2) {

```

Comment:
I copied formatting from other C++ samples. I prefer the above formatting so will reformat them all.